manifests/default-resources/internal-router/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+  - ../../internal-router
```

Comment:
is this file being used anywhere?

Reply:
not sure I even understand the purpose of the `default-resources` folder -- just did it the same as for other components